docs/go/quickstart/users-profile-sample-application.md

```diff
@@ -0,0 +1,179 @@
+---
+id: users-profile-sample-application
+title: Users Profile Sample Application
+sidebar_label: Gin-Mongo Sample 2
```

Comment:
`Gin-Mongo Sample 2` doesn't sound good, instead we can add another section named `Sample Applications` under which we can add **URL Shortner** and **Users Profile**.
Suggestions @nehagup @SanskritiHarmukh

Reply:
Yes what we can do is under quickstart we can have folders like Sample Go applications where we can have url shortner and users profile and inside Sample Java applications we can have Employee-Manager App. The sidebar label can be given accordingly.

Reply:
Since this is Go specific example I believe adding the sample application under Go Quickstart makes sense.

Reply:
We can create another issue for this, as this is not related to the current issue